hedera-node/src/main/java/com/hedera/services/state/migration/StateVersions.java

```diff
@@ -43,9 +43,10 @@ public final class StateVersions {
 	public static final int RELEASE_024X_VERSION = 17;
 	public static final int RELEASE_025X_VERSION = 18;
 	public static final int RELEASE_0260_VERSION = 19;
+	public static final int RELEASE_0270_VERSION = 20;
 
 	public static final int MINIMUM_SUPPORTED_VERSION = RELEASE_024X_VERSION;
-	public static final int CURRENT_VERSION = RELEASE_0260_VERSION;
+	public static final int CURRENT_VERSION = RELEASE_0270_VERSION;
```

Comment:
NOTE: This needs to be changed to reflect the release version we are targetting.